services/web-app/utils/schema.js

```diff
@@ -137,6 +156,22 @@ export const getLicense = (asset) => {
   return license
 }
 
+/**
+ * Gets the license from a designKit
+ * @param {import('../typedefs').DesignKit} designKit
+ * @returns {string} License
+ */
+export const getDesignKitLicense = (designKit) => {
+  const defaultLicense =
```

Comment:
not sure what's happening here 🤔 , what is the intent?

Reply:
Ooops ovrersight!